content/en/blog/2023/e2e-testing-best-practices.md

```diff
@@ -0,0 +1,414 @@
+---
+layout: blog
+title: "E2E Testing Best Practices, Reloaded"
+date: 2023-04-04
```

Comment:
For a publication date, how about 2023-04-12?

Reply:
Works for me, updated accordingly: https://github.com/kubernetes/contributor-site/compare/846fba3cd23f37924ea22e687ad015d838a4a7fd..7f1a7e9808047ed5c84b577428ccdfab209c4d4e